### PR TITLE
 Initial Support Oracle and refatoring

### PR DESCRIPTION
**Pull request by Yasara123**

### CODE REVIEW

File: components/org.wso2.carbon.status.dashboard.core/src/main/java/org/wso2/carbon/status/dashboard/core/dbhandler/DBHandler.java [hunk 1]
```diff
@@ -100,7 +100,7 @@ public boolean delete(PreparedStatement stmt) {
             stmt.close();
             return true;
         } catch (SQLException e) {
-            throw new RDBMSTableException(" Error while processing the dDELETE operation in " + stmt.toString(), e);
+            throw new RDBMSTableException(" Error while processing the dDELETE operation.", e);
```
Comment (gimantha) [inline components/org.wso2.carbon.status.dashboard.core/src/main/java/org/wso2/carbon/status/dashboard/core/dbhandler/DBHandler.java]: dDELETE?

File: components/org.wso2.carbon.status.dashboard.core/src/main/java/org/wso2/carbon/status/dashboard/core/dbhandler/StatusDashboardWorkerDBHandler.java [hunk 2]
```diff
@@ -296,15 +296,15 @@ private boolean insert(String columnNames, Object[] records, String tableName) t
...
+            throw new RDBMSTableException("Attempted execution of insert query produced an exceptions" +
                     " in " + DATASOURCE_ID, e);
         } finally {
             if (stmt != null) {
                 try {
                     stmt.close();
                 } catch (SQLException e) {
                     //ignore
-                    logger.error("Error closing statement at inser.", e);
+                    logger.error("Error closing statement at insert.", e);
```
Comment (gimantha) [inline components/org.wso2.carbon.status.dashboard.core/src/main/java/org/wso2/carbon/status/dashboard/core/dbhandler/StatusDashboardWorkerDBHandler.java]: Shall we add e.getMessage to the error?

File: components/org.wso2.carbon.status.dashboard.core/src/main/java/org/wso2/carbon/status/dashboard/core/dbhandler/DBHandler.java (hunk 1 above)
Comment (Yasara123) [reply components/org.wso2.carbon.status.dashboard.core/src/main/java/org/wso2/carbon/status/dashboard/core/dbhandler/DBHandler.java]: Fixed in 32f9bce1376e3d0a5f03e1723915ce3dd6c53df0

File: components/org.wso2.carbon.status.dashboard.core/src/main/java/org/wso2/carbon/status/dashboard/core/dbhandler/StatusDashboardWorkerDBHandler.java (hunk 2 above)
Comment (Yasara123) [reply components/org.wso2.carbon.status.dashboard.core/src/main/java/org/wso2/carbon/status/dashboard/core/dbhandler/StatusDashboardWorkerDBHandler.java]: Fixed in acaafe89c43697be9ef1532a725f8c6bdc4ae6d6

**Pull request merged**